artifacts/artifacts.go

```diff
@@ -8,6 +8,12 @@ import (
 	"github.com/NethermindEth/starknet.go/types"
 )
 
+//go:embed hello_starknet_compiled.casm.json
+var ExampleWorldCasm []byte
+
+//go:embed hello_starknet_compiled.sierra.json
+var ExampleWorldSierra []byte
+
 //go:embed account.json
 var AccountCompiled []byte
```

Comment:
As the old contract files are being removed it probably would be good to also remove some of the files from the artifacts that were only used by those files. This is one of them for example. Not sure if they could be used for the tests of the contract methods, but if they are, it would be just a matter of re-adding them

Reply:
Good point, turns out we don't need the artifacts pkg anymore, so I've deleted it